tests/test_metrics.py

```diff
@@ -95,20 +95,8 @@ def test_mark(self):
             k=test_k
         )
 
-
-    def test_pk(self):
-        """
-        Test precision@k computation
-        """
-
-        self.assertEqual(
-            metrics._pk(
-                actual = ['a', 'b', 'c'],
-                predicted = ['a', 'z', 'b'],
-                k = 3
-            ),
-            2/3
-        )
+        # THEN the mean absolute recall @ k should equal the expected value
+        self.assertEqual(mean_abs_recall_k, 0.25)
```

Comment:
This line got deleted in some weird diff / rebase issue situation

Reply:
oh that's my bad, thanks for the update